app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -187,6 +179,32 @@ class AvatarView @JvmOverloads constructor(
         )
     }
 
+    private fun handleDisplayType(
```

Comment:
Maybe we could name it something along the lines of `loadAvatarByDisplayType`? "handle" is pretty vague